src/CoreBundle/private/library/classes/dbobjects/TCMSTableConf.class.php

```diff
@@ -152,11 +154,19 @@ public function GetCmsTblListExtension($sExtensionName)
      */
     public function &GetFields(&$oTableRow, $loadDefaults = false, $bDoNotUseAutoObjects = false)
     {
+        $languageService = self::getLanguageService();
+        $currentLanguage = $languageService->getLanguage($this->GetLanguage());
```

Comment:
LanguageServiceInterface::getLanguage() expects a string argument, but $this->GetLanguage() might return null (although its signature states that it's always a string). We should be strict and handle this case explicitly.